### DISCUSSION TITLE
Git and GitHub Workflow | 1st Issue Ticket Task | Working with GitHub web UI

### DISCUSSION THREAD

**Discussion by Ifycode** [collabo-community/.github#4] [Git and GitHub Workflow "Good First Issue" Tickets]
#### Background

Hi there, welcome to @collabocate-osc! We have created this first (1st) issue ticket task assignment for anyone who:
- Is new to Git and GitHub and wants to learn how to work with them, especially while collaborating with others.
- Is looking for a platform to learn Git and GitHub, while dealing with repository/project that is owned/maintained by someone else.
- Would like [simple community tasks to understand @collabocate-osc's Git and GitHub workflow](https://github.com/orgs/collabocate-osc/projects/1?filterQuery=label%3A%22For%3A+Community+%5BGeneral%5D%22+label%3A%22topic%3A+good+first+issue%22+label%3A%22topic%3A+git+%26+github+workflow%22), before starting to contribute to community projects.

#

#### Helpful resource
This youTube playlist will give you an idea of how to work with GitHub web User Interface: https://youtube.com/playlist?list=PLMDhbo3xlD1ESa1_9WN4yG7bGDDksEQx7&si=F9lzacG1cr9WGS-M

#

#### Issue description

In this **1st task assignment**, you are required to create a file with GitHub markdown content **from the GitHub web User Interface**.  Find more information in the expected solution section below.

#

#### Expected solution

> [!TIP]  
> You are to create your branch, folder and file using the GitHub web User Interface.

- Create a new branch from the `develop` branch. Your branch name should be `@GH-replaceThisPartAfterTheHyphenWithTheIssueNumberForThisTicket` (don't add the # sign to the ticket number, and no space in there). It is in this new branch that you should make your changes in and submit from.
- Create a new folder called `@replace-this-text-with-your-own-GitHub-username` in the root of the repository that you are to do this task in and submit to (which repository is this? - see the additional context section below).
- Create a file called `gitcollab-ui-task.md` in the root of your newly created folder.
- Use the content of [the ui-task.md file found at the root of this `@gitcollab-learn` folder](https://github.com/collabocate-osc/use-me-for-experiments/blob/develop/%40gitcollab-learn/ui-task.md), in your own newly created file. Once you read the content of the ui-task.md file, you will know what text to change or replace.
- Before you commit your changes i.e. before you click on the "commit changes" button, take a screenshot. You will need it for your pull request, to show proof that you used the GitHub Web Interface. Take note: Before making the screenshot, expand your browser to full screen mode. Make sure to scroll to the top of the page so that the top of the page is not cropped out. Here's an example screenshot from a contributor: https://github.com/user-attachments/assets/70d8cd84-be9a-4651-932f-71505438aff9

#

#### Additional context
- Let us know in the comment of this issue ticket (GitHub discussion) that you will be participating in this 1st task assignment. Ensure to tag @collabocate-osc/maintainers in your comment. That way, maintainers will be notified and will be prepared to help review your pull request when you submit.
- Repo URL (where you are to do and submit this task to): https://github.com/collabocate-osc/good-first-issue-GIT-solutions
- If you have any questions or need any clarification when working on this ticket, please use the `#git-workflow-support` channel in our [discord community server](https://resources.collabocate.community).
- To help you better understand how to work with other contributors within @collabocate-osc better, we will review your pull request based on the directions given in this issue ticket and our [Public Documentation Resource Guide](https://resources.collabocate.community).

#

#### Testing checklist

Note (to contributor taking on this task assignment): Copy and paste the testing checklist below (without this note) to your pull request body description when submitting your fix. This will guide (you and) the contributors who will review your fix to know what is important to check or test for.
- [ ] My branch for this task assignment is created from the `develop` branch, and the name of my branch is a mix of `@GH-` and the ticket's issue number (no # sign and no space in there).
- [ ] The folder with its name having a mix of `@` and my GitHub user name has been created in the root of the `good-first-issue-GIT-solutions` repository.
- [ ] A file called `gitcollab-ui-task.md` is in the root of my newly created folder.
- [ ] The `gitcollab-ui-task.md` file uses the content of [the ui-task.md file found at the root of the `@gitcollab-learn` folder](https://github.com/collabocate-osc/use-me-for-experiments/blob/develop/%40gitcollab-learn/ui-task.md). I have also followed the directions in there to replace text where necessary.
- [ ] I have taken a screenshot to show that I used the GitHub Web Interface for this task, as specified in the issue ticket. I have added the screenshot to the screenshot section of my pull request's body description.
- [ ] I certify that I ran my checklist.

**Comment by Goodnews-code** [comment]
# Hi @collabo-community/maintainers
## I will be participating in this 1st task assignment.

**Reply by Ifycode** [reply]
👍🏼 @Goodnews-code 

> You don't need to format your comment as an header with markdown though. Maintainers don't get notified if done that way.

**Comment by BA-atomic** [comment]
Hi @collabo-community/maintainers
I'll be participating in this 1st task assignment

**Reply by Ifycode** [reply]
Welcome @BA-atomic 
Please go ahead 
> Reply is late because community is on break till 15 January.

**Comment by Yanmi01** [comment]
~~Hi @collabo-community/maintainers
I'll be participating in this first task assignment~~

**Reply by Ifycode** [reply]
👍🏼 @Yanmi01

**Comment by BA-atomic** [comment]


**Reply by Ifycode** [reply]
@BA-atomic since you've commented on this one before to take it on, you don't have to comment here again.
But you get to the other tickets for tasks 2, 3 etc that you have not commented before, you can resume commenting on the issue tickets before you start.
Anyway, go ahead.

**Reply by Ifycode** [reply]
You can just remove this duplicate comment you just added, so that we don't have 2 requests from you on the same issue ticket @BA-atomic

**Comment by OpeyemiAdeniji** [comment]
Hi @collabo-community/maintainers
I'll be participating in this first task assignment

**Reply by Ifycode** [reply]
Welcome 👍🏼

**Comment by KBandipo** [comment]
Hi @collabo-community/maintainers 
I will be participating in this 1st task assignment.

**Reply by Ifycode** [reply]
🎉 @KBandipo

**Comment by Danbaba1** [comment]
Hi @collabo-community/maintainers 
I will be participating in this 1st task assignment

**Reply by Ifycode** [reply]
🎉 @Danbaba1

**Comment by Oyebliss** [comment]
Hi @collabo-community/maintainers 
I will be participating in this 1st task assignment.

**Reply by Ifycode** [reply]
🎉 @Oyebliss

**Comment by KamiNation** [comment]
Hi, @collabo-community/maintainers. I will be participating in this first task assignment.

**Reply by Ifycode** [reply]
👍🏼

**Comment by elizabethobiagba** [comment]
I will be participating in this 1st task assignment @collabo-community/maintainers

**Reply by Ifycode** [reply]
👍🏼

**Comment by oluwadamilola21** [comment]
Hi @collabo-community/maintainers
I'll be participating in this 1st task assignment

**Reply by Ifycode** [reply]
👍🏼

**Comment by Ifycode** [comment]
> [!IMPORTANT]  
> The name of our community has changed from `collabo-community` to `collabocate-osc`. Please tag maintainers with @collabocate-osc/maintainers from now on. Thank you.

**Comment by Yanmi01** [comment]
Hi @collabocate-osc/maintainers,
I will be participating in this first task

**Reply by Ifycode** [reply]
👍🏼
